### DISCUSSION TITLE
Is it possible to run on DigitalOcean managed Kubebernetes?

### DISCUSSION THREAD

**Discussion by ccrvlh** [openebs/openebs#3527] [Q&A] [answered]
Hi, I've been trying to deploy EBS on my DO managed cluster, but no success.
I can install the helm and see the pods and the storage classes, but when trying to run the example for PV Device, the PVC is created, but not the PV, so the PVC and the pod keeps pending. I'm not sure, but it may be because DigitalOcean has a closed node, so no way to mount the disks and format them.
Wondering if there's a work around for this.
On the documentation it mentions SSH into the node to change the volume drivers, but when I tried that the Node died and didn't recover, not sure I was doing the right thing though, the docs are a bit vague in that aspect.
Any ideas?

**Comment by hrudaya21** [comment, marked as answer]
Can you remember what was the exact issue and what OpenEBS engine you have installed.
Currently We are actively working on Mayastor and We can install it in managed kubernetes engine.
Please refer to following documentation for more details.
https://mayastor.gitbook.io

**Comment by orville-wright** [comment]
Closing this discussion. No response from user @ccrvlh
